moveit_kinematics/CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ set(THIS_PACKAGE_INCLUDE_DIRS
 catkin_package(
   LIBRARIES
   INCLUDE_DIRS
-    ${EIGEN3_INCLUDE_DIRS}
     ${THIS_PACKAGE_INCLUDE_DIRS}
   CATKIN_DEPENDS
     pluginlib
     moveit_core
     moveit_ros_planning
+    DEPENDS Eigen3
```

Comment:
This doesn't work.
`pkg_search_module(EIGEN3 REQUIRED eigen3)` defines a variable `EIGEN3_INCLUDE_DIRS`, but
the `DEPENDS Eigen3` makes catkin look for `Eigen3_INCLUDE_DIRS`.

Please change this to (including the newline+indention)
```
DEPENDS
  EIGEN3
```